cmd/argocd/commands/cluster.go

```diff
@@ -377,6 +377,13 @@ argocd cluster rm cluster-name`,
 
 			for _, clusterSelector := range args {
 				clusterQuery := getQueryBySelector(clusterSelector)
+
+				// Point users wishing to remove in-cluster to the correct setting
+				if clusterQuery.Server == "https://kubernetes.default.svc" {
```

Comment:
There's probably already a constant for the incluster server url we should probably use that